test/unit/utils/EventManager.mocha.ts

```diff
@@ -0,0 +1,128 @@
+/// <reference path='../test.mocha.ts' />
+
+describe('n3Charts.Utils.EventManager', () => {
+  var eventMgr: n3Charts.Utils.EventManager = undefined;
+
+  beforeEach(() => {
+
+    eventMgr = new n3Charts.Utils.EventManager();
+  });
+
+  describe('init()', () => {
+    
+    it('should parse arguments', () => {
+
+      expect(() => eventMgr.init(['test'])).to.not.throwError();
+    });
+
+    it('should return the instance', () => {
+
+      var e = eventMgr.init([]);
+
+      expect(e).to.equal(eventMgr);
+    });
+  });
+
+  describe('on()', () => {
+    
+    it('should throw a TypeError, when the event is not initialized', () => {
+
+      var e = eventMgr.init(['test']);
+
+      expect(() => e.on('test', () => 'test')).to.not.throwError();
+      expect(() => e.on('other-test', () => 'test')).to.throwError(TypeError);
+    });
+
+    it('should return the instance', () => {
+
+      var e = eventMgr.init(['test']);
+
+      expect(e.on('test', null)).to.equal(eventMgr);
+    });
+  });
+
+  describe('trigger()', () => {
+  
+    it('should trigger the handler', () => {
+
+      var triggered = false;
+      var e = eventMgr.init(['test']);
+
+      e.on('test', () => triggered = true);
+      e.trigger('test');
+
+      expect(triggered).to.equal(true);
+    });
+
+    it('should trigger only last handler', () => {
+
+      var triggered1 = false;
+      var triggered2 = false;
+      var e = eventMgr.init(['test']);
+
+      e.on('test', () => triggered1 = true);
```

Comment:
to be honest, i have not a lot experience how and when to use `sinon.stub()` exactly (I mean I know why to use it), but especially when I want to check the execution order of sth it is simper for me to write a helper class. I would be glad if you could correct this for me (after the merge).